packages/wdio-mocha-framework/src/utils.ts

```diff
@@ -1,5 +1,8 @@
-export function loadModule (name, context) {
+import type { MochaContext } from './types'
+
+export function loadModule (name: string, context: MochaContext) {
     try {
+        // @ts-ignore
         module.context = context
```

Comment:
[_not relations to the PR_] Looks like some dark magic :)

Reply:
true, let's keep it as is and eventually clean this up.

Reply:
agree, no idea how it works